coral-hive/src/main/java/com/linkedin/coral/hive/hive2rel/CoralConvertletTable.java

```diff
@@ -20,10 +20,11 @@
 
 
 /**
- * ConvertletTable for Hive Operators
+ * ConvertletTable for transformations only relevant to Coral's Intermediate Representation, not specific
+ * any SQL dialect. These transformations keep data parity between the SqlNode and RelNode layer, keeping the IR intact.
  * @see ReflectiveConvertletTable documentation for method naming and visibility rules
  */
-public class HiveConvertletTable extends ReflectiveConvertletTable {
+public class CoralConvertletTable extends ReflectiveConvertletTable {
```

Comment:
I think we can move it to coral-common?

Reply:
There's a few newly "Coral IR specific" classes still in `coral-hive` such as `CoralINOperator` and `CoralRelToSqlNodeConverter`. The plan is to refactor these correctly in one future PR.